tests/test_run_app.py

```diff
@@ -685,6 +686,24 @@ async def on_startup(app):
     exc_handler.assert_called_with(patched_loop, msg)
 
 
+def test_run_app_keepalive_timeout(patched_loop):
+    new_timeout = 1234
+    base_runner_init_mock = mock.MagicMock()
+    base_runner_init_orig = BaseRunner.__init__
+
+    def base_runner_init_spy(self, *args, **kwargs):
+        base_runner_init_mock(*args, **kwargs)
+        base_runner_init_orig(self, *args, **kwargs)
+
+    with mock.patch.object(BaseRunner, "__init__", base_runner_init_spy):
+        app = web.Application()
+        web.run_app(app, keepalive_timeout=new_timeout,
+                    print=stopper(patched_loop))
+
+    base_runner_init_kwargs = base_runner_init_mock.call_args[1]
+    assert (base_runner_init_kwargs["keepalive_timeout"] == new_timeout)
```

Comment:
assert is not a function and so it does not require parentheses: 
```suggestion
    assert base_runner_init_kwargs["keepalive_timeout"] == new_timeout
```

Reply:
The assert used to be broken across two lines and the parentheses remained there.